tests/upgradePMM_test.js

```diff
@@ -178,6 +196,42 @@ if (iaReleased) {
   );
 }
 
+if (versionMinor >= 13) {
+  Scenario(
+    'Verify user has failed checks before upgrade @pre-upgrade @ami-upgrade @pmm-upgrade',
+    async ({
+      I, settingsAPI, databaseChecksPage, securityChecksAPI,
+    }) => {
+      const runChecks = locate('button').withText('Run DB checks');
```

Comment:
Can be this locator defined in `databaseChecksPage.js`?

Reply:
there is no need for it as this is an old locator, from < 16 version